share/boot_config.json

```diff
@@ -1,5 +1,5 @@
-/* QQVGA = 0, QVGA = 1, VGA =2 */
 {
+  "_comment": "QQVGA = 0, QVGA = 1, VGA = 2",
```

Comment:
Should be in a different Pull Request because it is not related to the goal of this PR

Reply:
Ah, I see. Just remove this commit from your Pull Request

Reply:
Thank you very much for your review.
I removed this commit.